postgres-types/Cargo.toml

```diff
@@ -15,7 +15,7 @@ derive = ["postgres-derive"]
 with-bit-vec-0_6 = ["bit-vec-06"]
 with-chrono-0_4 = ["chrono-04"]
 with-eui48-0_4 = ["eui48-04"]
-with-geo-types-0_6 = ["geo-types-06"]
+with-geo-types-0_7 = ["geo-types-0_7"]
```

Comment:
Dropping 0.6 support breaks backwards compatibility. These features were set up so that multiple versions could be supported concurrently.